net-dns/bind_utils/bind_utils-9.16.36.recipe

```diff
@@ -5,11 +5,11 @@ COPYRIGHT="Internet Systems Consortium, Inc. ('ISC')"
 LICENSE="MPL v2.0"
 REVISION="1"
 SOURCE_URI="ftp://ftp.isc.org/isc/bind9/$portVersion/bind-$portVersion.tar.xz"
-CHECKSUM_SHA256="20bf727559302c933475904847041916bb6c279680c170babc01a76998e80ad3"
+CHECKSUM_SHA256="508c94e8c9884f6075fa820a51efae04a1758dbdd157b01695ea6cd07049c221"
 SOURCE_DIR="bind-$portVersion"
 PATCHES="bind_utils-$portVersion.patchset"
 
-ARCHITECTURES="?all !x86_gcc2"
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
Not checked on 32bit secondary architecture?

Reply:
This should work on x86 - but only verified on x64.

Reply:
Builds OK on secondary architecture 32bit, PS how do you check if it's working as intended? Tried running `nslookup www.google.be` but getting a `connection timed out: no servers could be found`.

You could add `commandBinDir` etc in the recipe to install the binaries in `$prefix/bin` instead of `$binDir` on 32bit: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#commandbindir-and-commandsuffix

You can also create a `debugInfoPackage`: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#definedebuginfopackage

Reply:
Type: `nslookup www.google.be 4.2.2.2`

I'm only submitting only a x64 build for now.